src/schema/feeds.ts

```diff
@@ -311,11 +310,6 @@ export const typeDefs = /* GraphQL */ `
       """
       unreadOnly: Boolean = false
 
-      """
-      Force refresh the feed
-      """
-      refresh: Boolean = false
-
```

Comment:
@rebelchris just to double check that frontend does not send this in any way 🙏 (if you did not already)

Reply:
I'm only removing it in the next app release here:
https://github.com/dailydotdev/apps/pull/3687

Reply:
And extension, does it request it?

Reply:
Please contact me via email at crazysuperhulk@gmail.com so we can chat privately